### PR TITLE
MES-4409 | feat(ADI2): implement VehicleChecks component for ADI Part 2 category.

### PR DESCRIPTION
**Pull request by AndyPerryKainos**
## Description

- Implement VehicleChecks ( Tell Me Questions ) form ADI Part 2 category.
- Includes dependency: mes-test-schema@3.24.3

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

IMG_0175

IMG_0176

IMG_0177

IMG_0178

IMG_0179

### CODE REVIEW

**Review by szabikr — APPROVED** [review]

File: src/pages/waiting-room-to-car/cat-adi-part2/components/vehicle-checks-modal/__tests__/vehicle-checks-modal.cat-adi-part2.analytics.effects.spec.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  beforeEach(() => {
+    actions$ = new ReplaySubject(1);
+    effects = TestBed.get(VehicleChecksModalAnalyticsEffects);
+    analyticsProviderMock = TestBed.get(AnalyticsProvider);
+    store$ = TestBed.get(Store);
+  });
+
+  describe('vehicleChecksModalViewDidEnter$ effect', () => {
+    it('should call analytics.setCurrentPage', (done) => {
+      store$.dispatch(new testsActions.StartTest(12345, TestCategory.BE));
```
Comment (rossfauldsbjss) [inline src/pages/waiting-room-to-car/cat-adi-part2/components/vehicle-checks-modal/__tests__/vehicle-checks-modal.cat-adi-part2.analytics.effects.spec.ts]: TestCategory BE - should be ADI2

File: src/pages/waiting-room-to-car/cat-adi-part2/components/vehicle-checks-modal/vehicle-checks-modal.cat-adi-part2.page.module.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  imports: [
+    IonicPageModule.forChild(VehicleChecksCatADIPart2Modal),
+    EffectsModule.forFeature([
+      VehicleChecksModalAnalyticsEffects,
+    ]),
+    IonicModule,
+    ComponentsModule,
+  ],
+})
+export class VehicleChecksModalCatBEModule { }
```
Comment (rossfauldsbjss) [inline src/pages/waiting-room-to-car/cat-adi-part2/components/vehicle-checks-modal/vehicle-checks-modal.cat-adi-part2.page.module.ts]: VehicleChecksModalCatBEModule - should be ADI2

**Review by rossfauldsbjss — CHANGES_REQUESTED** [review]
2 references to non ADI2 categories.

**Comment by AndyPerryKainos** [comment]
> 2 references to non ADI2 categories.

@rossfauldsbjss Good spots, thanks. Fixed and pushed.

**Review by rossfauldsbjss — APPROVED** [review]
looks good

**Pull request merged**